tools/TimeFileMaker.py

```diff
@@ -306,20 +307,20 @@ def from_seconds(seconds, signed=False):
     Converts a number of seconds into a string time.
     '''
     sign = ('-' if seconds < 0 else '+') if signed else ''
-    seconds = abs(seconds)
+    full_centiseconds=round(abs(seconds) * 100)
```

Comment:
For consistency with surrounding code
```suggestion
    full_centiseconds = round(abs(seconds) * 100)
```